siriuspy/siriuspy/search/id_search.py

```diff
@@ -24,18 +20,48 @@ class IDSearch:
         'MANACA':   'SI-09SA:ID-APU22',
         'SABIA':    'SI-10SB:ID-EPU50',
         'IPE':      'SI-11SP:ID-APU58',
-        # 'PAINEIRA': 'SI-14SB:ID-WIG180',
-        # 'SAPUCAIA': 'SI-17SA:ID-???',
+        'PAINEIRA': 'SI-14SB:ID-WIG180',
     }
-    _idname2beamline = {v: k for k, v in _beamline2idname.items()}
 
-    _idname_2_orbitcorr = {}
+    _idname2beamline = {v: k for k, v in _beamline2idname.items()}
 
-    _idname_2_orbitffwd_fname = {}
+    _idname_2_idff = {
+        'SI-06SB:ID-APU22': dict(),
```

Comment:
maybe this default value could be None

Reply:
I used empty dict so that loops over pairs of keys/values would run inconsequentially. but we can abandon this option, which currently is not used.